tests/tests.js

```diff
@@ -147,7 +147,21 @@ var runTests = function runTests(arrayFrom) {
 		t.end();
 	});
 
-	test('no setters are called for indexes', { 'skip': !Object.defineProperty }, function (t) {
+	var defineProperty = (function () {
```

Comment:
also maybe make the test skip on both `!require('define-properties').supportsDescriptors` as well as this test, so it's explicit that there's two requirements?